frontend/src/components/BookCard.tsx

```diff
@@ -0,0 +1,26 @@
+import {Book} from "../models/Book";
+
+type Props = {
+    book: Book
+}
+export default function BookCard(props: Props) {
+    return (
+        <div>
+            <div className={"book-card"}>
+                {props.book.img ? <img src={props.book.img} alt="Book"/> : <img alt="Book"/>}
```

Comment:
Is the alternative image missing? There is only an alt attribute in the second image tag.